src/index.html

```diff
@@ -4,9 +4,392 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header 
+      class="header"
+    >
+      <a href="#home" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo-image"
+        >
+      </a>
+      <nav 
+        class="nav"
+      >
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              href="#Apple"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a 
+              href="#Samsung" 
+              class="nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-item">
+            <a 
+              href="#Smartphones" 
+              class="nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#Laptops&Computers"
+              class="nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav-item">
+            <a 
+              href="#Gadgets" 
+              class="nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav-item">
+            <a 
+              href="#Tablets" 
+              class="nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav-item">
+            <a 
+              href="#Photo" 
+              class="nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav-item">
+            <a 
+              href="#Video" 
+              class="nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div 
+          class="card catalog__card"
+          data-qa="card"
+        >
+          <img 
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__image"
+          >
+          <p class="card__label">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```

Comment:
```suggestion
          <h3 class="card__label">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```
It can be h3/4 as it's a title of card